### PR TITLE
SISRP-22116 Start splitting out My Campus Links for exStudent role

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.berkeley.edu/browse/SISRP-22116

Up to now, the server has just been returning whatever the "student" role got.

### CODE REVIEW

File: public/json/campuslinks.json [hunk 1]
```diff
@@ -728,14 +728,14 @@
...
-          "subcategory": "Points of Interest"
+          "subcategory": "Athletics"
         }
       ]
     },
     {
-      "name": "Cal Performances",
-      "description": "Information and tickets for Cal music, dance, and theater performances",
-      "url": "http://www.calperformances.org/",
+      "name": "CalDining",
```
Comment (johncrossman) [inline public/json/campuslinks.json]: @raydavis Do we care that, officially, the name "Cal Dining" has a space? :)

Comment (raydavis) [inline public/json/campuslinks.json]: Up to now we've been a callous bunch, but you've tweaked my conscience. Amendment coming...

**Comment by raydavis** [comment]
Space given.

**Comment by johncrossman** [comment]
Space received. Thx, @raydavis  Waiting on a happy Travis.

**Pull request merged**